### PR TITLE
fix: filename defaults

### PR DESCRIPTION
**Pull request by davecharles**
- removed mutable default args.
- fixed broken test.

### CODE REVIEW

File: pii_secret_check_hooks/check_file/base_content_check.py [hunk 1]
```diff
@@ -39,8 +39,9 @@ def __init__(
         self,
         check_name,
         allow_changed_lines=False,
-        excluded_file_list=[],
+        excluded_file_list=None,
```
Comment (rossmiller) [inline pii_secret_check_hooks/check_file/base_content_check.py:42]: Good catch! Could we do the assignment on the self var and save some lines of code?

File: pii_secret_check_hooks/check_file/file_content.py [hunk 2]
```diff
@@ -42,11 +42,12 @@ class CheckFileContent(CheckFileBase):
     def __init__(
         self,
         allow_changed_lines=False,
-        excluded_file_list=[],
-        custom_regex_list=[],
+        excluded_file_list=None,
+        custom_regex_list=None,
     ):
+        excluded_file_list = [] if excluded_file_list is None else excluded_file_list
+        custom_regex_list = [] if custom_regex_list is None else custom_regex_list
```
Comment (rossmiller) [inline pii_secret_check_hooks/check_file/file_content.py:49]: Same as above.

File: pii_secret_check_hooks/check_file/ner.py [hunk 3]
```diff
@@ -39,10 +39,12 @@ class CheckForNER(CheckFileBase):
...
         self,
         allow_changed_lines=False,
-        excluded_file_list=[],
-        excluded_ner_entity_list=[],
+        excluded_file_list=None,
+        excluded_ner_entity_list=None,
         ner_output_file=None,
     ):
+        excluded_file_list = [] if excluded_file_list is None else excluded_file_list
+        excluded_ner_entity_list = [] if excluded_ner_entity_list is None else excluded_ner_entity_list
```
Comment (rossmiller) [inline pii_secret_check_hooks/check_file/ner.py:47]: Same as above.

File: pii_secret_check_hooks/pii_secret_file_content.py [hunk 4]
```diff
@@ -20,13 +20,13 @@ def main(argv=None):
     parser.add_argument(
         "--exclude",
         nargs="?",
-        default="pii-secret-exclude.txt",
```
Comment (rossmiller) [inline pii_secret_check_hooks/pii_secret_file_content.py:23]: Sorry! Just to be a PIA I want to use the non '.' version of the file name,

File: tests/check_file/test_base_content.py [hunk 5]
```diff
@@ -43,7 +43,7 @@ def test_log_updated(load_json):
     check_base = CheckFileBaseTest(
         check_name="test_base",
     )
-    assert "tests/assets/test.txt" in check_base.log_data["files"]
```
Comment (rossmiller) [inline tests/check_file/test_base_content.py:46]: I'm confused by this one. Maybe we should have a chat?

**Review by rossmiller — CHANGES_REQUESTED** [review]
Thanks for this! Might be worth a chat about a few points.

File: pii_secret_check_hooks/pii_secret_file_content.py (hunk 4 above)
Comment (davecharles) [reply pii_secret_check_hooks/pii_secret_file_content.py]: Cool

**Review by rossmiller — APPROVED** [review]
I made a couple of minor changes but looks good!

**Pull request merged**